src/js/components/utils/detailPageLayout.tsx

```diff
@@ -35,8 +38,40 @@ const DetailPageLayout = ({
   sidebar?: ReactNode;
   children?: ReactNode;
   image?: string;
+  type?: 'epic' | 'task';
 }) => {
   const showHeaderImage = Boolean(image && !description);
+  let popover = null;
+  const heading = i18n.t('Navigation breadcrumb');
+
+  popover =
+    type === OBJECT_TYPES.EPIC ? (
```

Comment:
@jgerigmeyer Moved the breadcrumb popover here. Why isn't the text changing? Am I referencing `OBJECT_TYPES` incorrectly?

Reply:
You were _accepting_ a `type` prop here, but not passing one in from where it's used. Fixed in https://github.com/oddbird/Metecho/commit/56cfb220d46b81aade5d2123aebe659e9a51d7da

Reply:
Aha. Got it. Thanks!